modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -36,6 +37,51 @@
+* A new config option `session_forced_logout_timeout` has been introduced in `config.php`. For advanced security, this option can be used to expire user sessions a configurable amount of seconds after they close the ownCloud browser tab or the whole browser, respectively, forcing the the user to log in again when they access ownCloud the next time. This option is disabled by default.
+
+=== Notable changes
+
+* The feature _Resend invitation mail_ in user management does not apply to guest users anymore as those have a different invitation flow than regular users (user-based vs. admin-based). https://github.com/owncloud/core/pull/40032[#40032]
+* Group administrators will now only see the groups they are an administrator of in user management (previously they also so other groups but were unable to manage them). https://github.com/owncloud/core/pull/39752[#39752]
+* ownCloud Server 10.9 introduced a new feature to xref:server_release_notes.adoc#highly-improved-initial-sync-and-discovery-performance[improve initial sync and discovery performance] which has been enabled by default. As there have been performance issues in certain environments, 10.10 disables the feature by default. It is recommended to enable the feature based on evaluations with test systems. https://github.com/owncloud/core/pull/40016[#40016]
+* Storage encryption: Restoring a received shared file now also restores its versions correctly. https://github.com/owncloud/core/pull/39822[#39822]
+* Storage encryption: Moving a file out of a share now also takes care of versions correctly (previously they were corrupted under certain circumstances). https://github.com/owncloud/core/pull/39829[#39829]
+* The external storage administration user interface has been improved to avoid unnecessary credential exposure https://github.com/owncloud/core/pull/39841[#39841] https://github.com/owncloud/core/pull/39935[#39935]
+* The mail server configuration user interface has been improved to avoid unnecessary credential exposure https://github.com/owncloud/core/pull/39833[#39833]
+* The 'External Storage' feature (`files_external`) can now be disabled. https://github.com/owncloud/core/pull/39856[#39856]
+* General performance and stability improvements have been made. https://github.com/owncloud/core/pull/39847[#39847] https://github.com/owncloud/core/pull/40031[#40031]
+* The File Drop view for Upload-only public links has been visually improved to provide a better user experience. https://github.com/owncloud/core/pull/39900[#39900]
+* ownCloud Web: Guest users will now be explicitly listed as "Guests" (instead of "User") https://github.com/owncloud/core/pull/40013[#40013]
+
+=== Updated app versions
+
+TBC
+
+=== For developers
```

Comment:
Maybe also mention 
"Enhancement - Expose user type of share receiver in share api: [#40013](https://github.com/owncloud/core/pull/40013)"
?

That is another addition to data returned in an API response.